src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -861,6 +863,68 @@ GUI_TEST_CLASS_DEFINITION(test_7388) {
                   "2. Unexpected clipboard text: " + clipboardText2);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7390) {
+    // 1. Set SPAdes to any file
+    // Expected: SPAdes is invalid
+    class SetSpades : public CustomScenario {
+        void run(GUITestOpStatus& os) override {
+            AppSettingsDialogFiller::openTab(os, AppSettingsDialogFiller::ExternalTools);
+
+            QString toolPath = dataDir + "samples/FASTA/human_T1.fa";
+
+            AppSettingsDialogFiller::setExternalToolPath(os, "SPAdes", QFileInfo(toolPath).absoluteFilePath());
+            CHECK_SET_ERR(!AppSettingsDialogFiller::isExternalToolValid(os, "SPAdes"), 
+                          "SPAdes is expected to be invalid, but in fact it is valid");
+
+            GTUtilsDialog::clickButtonBox(os, GTWidget::getActiveModalWidget(os), QDialogButtonBox::Ok);
+        }
+    };
+
+    GTUtilsDialog::waitForDialog(os, new AppSettingsDialogFiller(os, new SetSpades()));
+    GTMenu::clickMainMenuItem(os, { "Settings", "Preferences..." }, GTGlobals::UseMouse);
+
+    // 2. Open WD
+   GTUtilsWorkflowDesigner::openWorkflowDesigner(os);
+
+    using TrimmomaticAddSettings = QPair<TrimmomaticDialogFiller::TrimmomaticSteps, QMap<TrimmomaticDialogFiller::TrimmomaticValues, QVariant>>;
+    QList<TrimmomaticAddSettings> steps;
+    steps.append(TrimmomaticAddSettings(TrimmomaticDialogFiller::TrimmomaticSteps::ILLUMINACLIP, {} ));
+    TrimmomaticDialogFiller* filler = new TrimmomaticDialogFiller(os, steps);
```

Comment:
nit: type is typed twice in the line (use auto)

Reply:
The line's been removed